tools/serverpod_cli/bin/util/promt_to_update.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:yaml/yaml.dart';
+
+const promtMessage = '''
+┌───────────────────────────────────────────────────────────────────────────────────────┐
+│ A new version of Serverpod is available!                                              │
+│                                                                                       │
+│ To update to the latest version, run "dart pub global activate serverpod_cli".        │
+│ Also, do not forget to update packages in your server, client, and flutter projects.  │
+└───────────────────────────────────────────────────────────────────────────────────────┘
+''';
+
+/// Check current Serverpod CLI version and promt user to update if needed
+Future<void> promtToUpdateIfNeeded() async {
```

Comment:
Hope we can give the latest version in the cli for better info.

```console
A new version-1.0.1 of Serverpod is available!
```

something like this. LMK if this is okie, so I can share the rest info(API Endpoints) of pub.dev.

Reply:
Or How about writing a command/args like `serverpod upgrade` - to upgrade the serverpod cli version and `serverpod update` (in the project) to update the serverpod deps in the project?

Reply:
If possible, try [this](https://sourceb.in/FHYPUpPdcK)

Reply:
It's a good idea. But I'm not sure about updating dependencies. Because a user can call `serverpod upgrade` out of the workspace.